dencam.py

```diff
@@ -53,8 +53,10 @@ def cleanup(flags):
     try:
         recorder = Recorder(configs)
         state = State(NUM_STATES)
+        Airplanemode = AirplaneMode(configs)
         button_handler = ButtonHandler(recorder,
                                        state,
+                                       Airplanemode,
```

Comment:
I am kind of wondering whether we should pass configs to the ButtonHandler constructor and create the AirplaneMode object there since ButtonHandler is truly the only client of this object?